pyanaconda/ui/gui/spokes/user.py

```diff
@@ -255,105 +260,156 @@ def initialize(self):
+        self._checker = input_checking.PasswordChecker(initial_password_content = self.password,
+                                                       initial_password_confirmation_content = self.password_confirmation,
+                                                       policy = input_checking.get_policy(self.data, "user"))
+        # configure the checker for password checking
+        self.checker.name_of_password = _(constants.NAME_OF_PASSWORD)
+        self.checker.name_of_password_plural = _(constants.NAME_OF_PASSWORD_PLURAL)
+        self.checker.username = self.username
+        # remove any placeholder texts if either password or confirmation field changes content from initial state
+        self.checker.password.changed_from_initial_state.connect(self.remove_placeholder_texts)
+        self.checker.password_confirmation.changed_from_initial_state.connect(self.remove_placeholder_texts)
+        # connect UI updates to check results
+        self.checker.checks_done.connect(self._checks_done)
+
+        # username and full name checks
+        self._username_check = input_checking.UsernameCheck()
+        self._fullname_check = input_checking.FullnameCheck()
+        # empty username is considered a success so that the user can leave
+        # the spoke without filling it in
+        self._username_check.success_if_username_empty = True
+        # check that the password is not empty
+        self._empty_check = input_checking.PasswordEmptyCheck()
+        # check that the content of the password field & the conformation field are the same
+        self._confirm_check = input_checking.PasswordConfirmationCheck()
+        # regards both fields empty as success to let the user escape
+        self._confirm_check.success_if_confirmation_empty = True
+        # check password validity, quality and strength
+        self._validity_check = input_checking.PasswordValidityCheck()
+        # connect UI updates to validity check results
+        self._validity_check.result.password_score_changed.connect(self.set_password_score)
+        self._validity_check.result.status_text_changed.connect(self.set_password_status)
+        # check if the password contains non-ascii characters
+        self._ascii_check = input_checking.PasswordASCIICheck()
+
+        # register the individual checks with the checker in proper order
+        # 0) is the username and fullname valid ?
+        # 1) is the password non-empty ?
+        # 2) are both entered passwords the same ?
+        # 3) is the password valid according to the current password checking policy ?
+        # 4) is the password free of non-ASCII characters ?
+        self.checker.add_check(self._username_check)
+        self.checker.add_check(self._fullname_check)
+        self.checker.add_check(self._empty_check)
+        self.checker.add_check(self._confirm_check)
+        self.checker.add_check(self._validity_check)
+        self.checker.add_check(self._ascii_check)
```

Comment:
The same as in root spoke. Could you please move the checks setup to the separate method.